nntrainer/tensor/tensor.cpp

```diff
@@ -1005,6 +1005,24 @@ int Tensor::add_i(Tensor const &m, float const alpha) {
   return ML_ERROR_NONE;
 }
 
+int Tensor::add_i_partial(unsigned int len, unsigned int addr_idx, Tensor &m,
+                          unsigned int incX, unsigned int incY,
+                          const Tensor alphas, unsigned int alpha_idx) {
+  if (dim.getDataType() == ml::train::TensorDim::DataType::FP32) {
+    saxpy(len, alphas.getValue<float>(alpha_idx), m.getData<float>(), incX,
+          getAddress<float>(addr_idx), incY);
+  } else if (dim.getDataType() == ml::train::TensorDim::DataType::FP16) {
+#ifdef ENABLE_FP16
+    saxpy(len, alphas.getValue<_FP16>(alpha_idx), m.getData<_FP16>(), incX,
```

Comment:
2. Acceleration : Use SIMD haxpy in fp16 case. Previously it was only using naive loop.

In your comment, you mentioned that you are using haxpy, do you mean saxpy? 
Or is SIMD haxpy equivalent to saxpy?

Reply:
It will internally use haxpy in `blas_neon.cpp` but the interface function of using that is named as `saxpy`.
This needs fixing in the near future